sample-android/build.gradle.kts

```diff
@@ -17,7 +17,7 @@
 plugins {
     id("com.android.application")
     id("kotlin-android")
-    id("com.getkeepsafe.dexcount")
+//    id("com.getkeepsafe.dexcount")
```

Comment:
Nitpick: Is that any longer needed?

Reply:
it is, but need to wait for an update of the library

Reply:
Ok